staff/nicolas-martin/e-commerce-protos/scripts/cart.js

```diff
@@ -0,0 +1,51 @@
+function Cart() {
```

Comment:
Cool implementation with the reduce method!!! congrats!

Reply:
thankssss! 🍡